cmd/ctr/shim.go

```diff
@@ -214,10 +216,18 @@ var shimExecCommand = cli.Command{
 		if err != nil {
 			return err
 		}
+
+		// read spec file and extract Any object
+		spec, err := ioutil.ReadFile(context.String("spec"))
```

Comment:
Could also be a good separate PR to mirror `run.go` on how the `Process` information is generated for execs